### DISCUSSION TITLE
@milkdown/preset-gfm documentation has wrong information?

### DISCUSSION THREAD

**Discussion by matiadev** [Milkdown/milkdown#139] [Q&A] [answered]
Hey! :wave:

I'm confused about the [@milkdown/preset-gfm](https://www.npmjs.com/package/@milkdown/preset-gfm) documentation because it's the same as [@milkdown/preset-commonmark](https://www.npmjs.com/package/@milkdown/preset-commonmark).

I was looking at the [NES example](https://codesandbox.io/s/milkdown-theme-nes-b0zmy) which works on CodeSandbox, but if you try using a local setup you have to install `@milkdown/preset-commonmark` otherwise you encounter an error.

```shell
> node_modules/@milkdown/plugin-tooltip/lib/utility/input.js:3:40: error: Could not resolve "@milkdown/preset-commonmark" (mark it as external to exclude it from the bundle)
    3 │ import { ModifyImage, ModifyLink } from '@milkdown/preset-commonmark';
```

Thanks for your great work! :smile:

**Comment by Saul-Mirone** [comment, marked as answer]
The documentation of preset-gfm is wrong, I need to update it.

And the preset-commonmark it the dependency of preset-gfm, if you install preset-commonmark I think it should be installed in node_modules, except for pnpm, because it's [organized by a different pattern](https://dev.to/zkochan/flat-nodemodules-is-not-the-only-way-mo2), I will try to fix it. Do you use pnpm?

**Comment by matiadev** [comment]
Thanks for confirming. :smile:

I use `npm`. I exported the example from CodeSandbox, and installed dependencies using `npm i`. For some reason that doesn't install the dependency, so `npm start` doesn't work. After I did `npm i @milkdown/preset-commonmark` it works as expected.
